app/webpacker/components/wca/FormBuilder/input/MarkdownEditor.js

```diff
@@ -100,8 +108,8 @@ function getOptions(disabled) {
     spellChecker: false,
     promptURLs: true,
     previewRender,
-    status: ['upload-image'],
-    uploadImage: true,
+    status: [],
```

Comment:
This list can be built using `[enableImageUpload && 'upload-image']` and `filter(Boolean)`

Reply:
Done